<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConfig</class>
 <widget class="QDialog" name="DlgConfig">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressources.qrc">
    <normaloff>:/GFX/config.png</normaloff>:/GFX/config.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbAutoStart">
     <property name="text">
      <string>(TODO)Launch on System start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbUnit">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Unit:</string>
     </property>
     <widget class="QRadioButton" name="rbByte">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>135</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Byte</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbBits">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>135</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bits</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
